src/js/components/App.js

```diff
@@ -20,8 +34,30 @@ export default class App {
     });
   }
 
-  setState({ lottoTickets }) {
-    this.lottoTickets = lottoTickets;
-    this.purchasedLotto.setState({ lottoTickets: this.lottoTickets });
+  updateWinningNumber(winningNumber) {
+    this.setState({ winningNumber });
+  }
+
+  onShowModal() {
+    this.resultModal.showModal();
+  }
+
+  onRestart() {
+    this.setState({ lottoTickets: [], winningNumber: {} });
+    this.purchaseAmountInput.reset();
+  }
+
+  setState({ lottoTickets, winningNumber }) {
+    if (lottoTickets) {
+      this.lottoTickets = lottoTickets;
+      this.purchasedLotto.setState({ lottoTickets: this.lottoTickets });
+      this.winningNumberInput.setState({ isVisible: lottoTickets.length > 0 ? true : false });
```

Comment:
`lottoTickets.length > 0` 이 자체로 boolean 아닐까요?ㅎㅎㅋㅋ

```suggestion
      this.winningNumberInput.setState({ isVisible: lottoTickets.length > 0 });
```

Reply:
하하하,,,(민-망) 앞으로는 요런실수는 없도록 주의하겠습니다...!!! 감사합니다!! 👍👍👍